CHANGELOG.md

```diff
@@ -1,4 +1,35 @@
 ## 5.13.0 (Unreleased)
+NOTES:
+* cloudbuildv2: `google_cloudbuildv2_repository` now uses mmv1 engine instead of DCL ([#16969](https://github.com/hashicorp/terraform-provider-google/pull/16969))
+* gkehub: updated beta api endpoint from v1beta1 to v1beta ([#16966](https://github.com/hashicorp/terraform-provider-google/pull/16966))
```

Comment:
```suggestion
```

Reverting https://github.com/GoogleCloudPlatform/magic-modules/pull/9846

Reply:
Note, this probably shouldn't have been a release note in GA anyway